go.mod

```diff
@@ -144,3 +144,5 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gotest.tools/v3 v3.1.0 // indirect
 )
+
+replace github.com/chainguard-dev/go-apk v0.0.0-20230906161245-0728258ab917 => github.com/thesayyn/go-apk v0.0.0-20230928231027-0ed5c9ad96a8
```

Comment:
Is there a go-apk PR that corresponds to this?

Reply:
oh yes, https://github.com/chainguard-dev/go-apk/pull/125